java/integrated-windows-authentication/README.md

```diff
@@ -0,0 +1,33 @@
+# Integrated Windows Authentication
+
+Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![Integrated windows authentication App](integrated-windows-authentication.png)
+
+## Use case
+
+IWA, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+
+## How to use the sample
+
+1. Enter the URL to your IWA-secured portal in the edit text view at the top of the screen.
+2. Tap either the "Search Public" (which will search for portals on www.arcgis.com) or "Search Secure" (which will search your IWA-secured portal), for web maps stored on the portal.
+3. If you tap "Search Secure", you will be prompted for a username (including domain, ie username@DOMAIN), and password.
```

Comment:
Besides `username@DOMAIN`, `domain\\username` is also accepted. Maybe "ie" should be "such as".